tests/safety/test_ford.py

```diff
@@ -82,6 +77,10 @@ class TestFordSafety(common.PandaSafetyTest):
   MAX_CURVATURE_DELTA = 0.002
   CURVATURE_DELTA_LIMIT_SPEED = 10.0  # m/s
 
+  ANGLE_DELTA_BP = [5., 25., 25.]
+  ANGLE_DELTA_V = [0.0002, 0.0001, 0.0001]  # windup limit
+  ANGLE_DELTA_VU = [0.000225, 0.00015, 0.00015]  # unwind limit
```

Comment:
use up/down for consistency

Reply:
https://github.com/commaai/panda/pull/1390